project/FrontEnd/collaborative_science_platform/lib/screens/node_details_page/widgets/proof_list_view.dart

```diff
@@ -38,9 +47,31 @@ class ProofListView extends StatelessWidget {
                     //   style: TextStyles.title4,
                     //   textAlign: TextAlign.start,
                     // ),
-                    TeXView(
-                        renderingEngine: const TeXViewRenderingEngine.katex(),
-                        child: TeXViewDocument(utf8.decode(proof[index].proofContent.codeUnits))),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: [
+                        SizedBox(
+                            width: 180,
+                            child: AppButton(
+                                text: showAnnotations ? "Show Text" : "Show Annotations",
```

Comment:
There are 'Show Annotation' button for each proof but actually they should be all in same state, I think moving these buttons from each proof card to the only one to top of them is more clear design but this version also have advantages to change it while in bottom of the page. So we can discuss it with frontend team later.